### DISCUSSION TITLE
Search for a user based on ORCID

### DISCUSSION THREAD

**Discussion by phochste** [community/community#125647] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Github supports personalizing a profile with an ORCID identifier [link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile). But, I see no way to search users based on ORCID. Do I miss something or is this feature not available?

**Comment by MrShadowRIFAT** [comment, marked as answer]
Hi there,

GitHub does indeed allow users to link their ORCID (Open Researcher and Contributor ID) to their profiles, which helps in authenticating their contributions and associating them with their academic and professional identities. This is particularly useful for researchers and academics who want to display their ORCID on their GitHub profiles.

However, as of now, GitHub does not provide a built-in feature to search for users based on their ORCID identifier directly. The search functionality on GitHub primarily focuses on repositories, code, issues, pull requests, and users by their username or profile information, but it does not include a filter or search parameter specifically for ORCID IDs.

If searching by ORCID is a feature you find valuable, you might consider submitting a feature request to GitHub's support or feedback channels. User feedback plays a crucial role in the development and enhancement of platform features.

For now, if you need to find a specific user's GitHub profile and you know their ORCID, you might need to look up their ORCID record and see if they have linked their GitHub profile there, or manually search for their name on GitHub.

If you have any other questions or need further assistance, feel free to ask!

Best regards,
MrShadowRIFAT
